third_party/proton/proton/viewer.py

```diff
@@ -189,8 +194,8 @@ def main():
         help="""List available metrics. Metric names are case insensitive and ignore units.
 Derived metrics can be created when source metrics are available.
 - time/s, time/ms, time/us, time/ns: time
-- avg_time/s, avg_time/ms, avg_time/us, avg_time/ns: time / kernel_exec_count
-- flop/s, gflop/s, tflop/s: flops / time
+- avg_time/s, avg_time/ms, avg_time/us, avg_time/ns: time / count
+- flop<8/16/32/64>/s, gflop<8/16/32/64>/s, tflop<8/16/32/64>/s: flops / time
```

Comment:
flop/s should still be supported.
The point is that users can provide either `flop<numeric>/s or flop/s`

Reply:
Fixed that